db/seeds.rb

```diff
@@ -0,0 +1,53 @@
+# This file should ensure the existence of records required to run the application in every environment (production,
+# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Example:
+#
+#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
+#     MovieGenre.find_or_create_by!(name: genre_name)
+#   end
+# db/seeds.rb
+
+# Clear existing data
+User.destroy_all
+
+# Create 4 users
+users = []
+4.times do |i|
+  user = User.new(
```

Comment:
**OPTIONAL**- You did have done well so far, but your seed file gives some errors, please fix this 👍 👍 
<img width="696" alt="rseryaery" src="https://github.com/ometman/budget-app/assets/73607512/e99c68c6-89b6-4963-9bdb-d3cbb8fde51b">